Automattic/newspack-theme
# WooCommerce: improvements to product page

### All Submissions:


### Changes proposed in this Pull Request:

This PR makes some general improvements to the single product pages for WooCommerce, including widening the content area, tidying up tables, and styling the tabs. 

Closes #1244, #1240.

### How to test the changes in this Pull Request:

1. Set up a few products; it should include a combination of Newspack's default products (with the group of Donation products), and at least one with the One Page Checkout plugin set up on them.
2. Apply the PR and run `npm run build`.
3. View the products; confirm they're full-width, that the tabs look more tab-like, that the 'name your price' section on the Donation product is tidied up, and nothing looks too weird:

<details>
<summary><strong>Regular Product</strong></summary>

**Before**

![screencapture-build-newspack-test-product-test-product-2021-02-26-17_41_54](https://user-images.githubusercontent.com/177561/109371469-f010fe00-7859-11eb-8577-7182dedb9112.png)

**After**
_Note: the 'current' tab should pick up the primary colour._

![screencapture-build-newspack-test-product-test-product-2021-02-26-17_40_55](https://user-images.githubusercontent.com/177561/109371471-f3a48500-7859-11eb-8461-b36b8f4d88ed.png)
</details>

<details>
<summary><strong>Add to Cart message</strong></summary>

**Before:**

![add-to-cart-before](https://user-images.githubusercontent.com/177561/109371598-8c3b0500-785a-11eb-9a06-1fb9ec880531.jpg)

**After:**

![add-to-cart-after](https://user-images.githubusercontent.com/177561/109371607-91984f80-785a-11eb-9faa-9a13ec58450c.jpg)

</details>

<details>
<summary><strong>Product with Name Your Price</strong></summary>

**Before**

![screencapture-build-newspack-test-product-donate-monthly-2021-02-26-17_37_28](https://user-images.githubusercontent.com/177561/109371407-a88a7200-7859-11eb-8f20-5767046b93e8.png)

**After**

![screencapture-build-newspack-test-product-donate-monthly-2021-02-26-17_39_26](https://user-images.githubusercontent.com/177561/109371410-ac1df900-7859-11eb-8c60-f9cc67c4a213.png)

</details>

<details>
<summary><strong>Product Group with Name Your Price</strong></summary>

**Before**

![screencapture-build-newspack-test-product-donate-2021-02-26-17_35_19](https://user-images.githubusercontent.com/177561/109371298-29953980-7859-11eb-8688-7bdbfa488ead.png)

**After**

![screencapture-build-newspack-test-product-donate-2021-02-26-17_36_02](https://user-images.githubusercontent.com/177561/109371302-2d28c080-7859-11eb-9873-45f9b380d523.png)

</details>

<details>
<summary><strong>Product with One Page Checkout</strong></summary>

**Note:** Some details -- like the checkout form, and the radio alignment for shipping -- also exist in the /cart page, and will be fixed with that page's fixes.

**Before**

![screencapture-build-newspack-test-product-example-digital-product-2021-02-26-17_31_42](https://user-images.githubusercontent.com/177561/109371169-88a67e80-7858-11eb-87c2-7ec8e0688b07.png)

**After**

![screencapture-build-newspack-test-product-example-digital-product-2021-02-26-17_30_12](https://user-images.githubusercontent.com/177561/109371173-952ad700-7858-11eb-85d2-8b384ed4bd9b.png)
</details>

<details>
<summary><strong>Product with AMP enabled</strong></summary>

Products will work better if AMP is disabled for them, but if it's not, they should still look good -- tabs should be hidden (since they don't work), and the message you get when adding to cart should look okay. 

**Before**

![screencapture-build-newspack-test-product-test-product-2021-02-27-12_47_33](https://user-images.githubusercontent.com/177561/109399835-fc4c9800-78f9-11eb-89fa-a3b12abc8000.png)

**After**
![screencapture-build-newspack-test-product-test-product-2021-02-27-12_44_41](https://user-images.githubusercontent.com/177561/109399808-d9ba7f00-78f9-11eb-9429-1e569bd81bdc.png)

</details>


### Other information:
